win32ss/user/user32/windows/cursoricon.c

```diff
@@ -72,6 +72,20 @@ ReadMemoryPng(
     state->currentPos += length;
 }
 
+/* libpng.dll is delay-loaded. If no libpng.dll exists, we have to avoid exception */
+static BOOL
+LibPngExists(VOID)
+{
+    static BOOL bLibPngFound = -1;
+    if (bLibPngFound == -1)
+    {
+        HINSTANCE hLibPng = LoadLibraryExW(L"libpng.dll", NULL, LOAD_LIBRARY_AS_DATAFILE);
```

Comment:
It would make sense to use delay-loading import for the dll (in user32 cmakefile), and doing this loadlibraryex call here, but then perhaps without the freelibrary.